src/helpers/urlIsAvailable.ts

```diff
@@ -0,0 +1,12 @@
+export const urlIsAvailable = async (path: string): Promise<boolean> => {
```

Comment:
bit unusual Promise usage...

What if?.. Something like this:

const myPromise = new Promise((resolve, reject) => {
  try {
    const response = await fetch(path)
    if (response.headers.get('content-type') === 'image/jpeg') {
      resolve(true);
    } else {
      reject(false);
    }
  } catch (err) {
    reject(err);
  }
});

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise

Reply:
Thanks, I fixed it in commit e9860952613603eb2cf3095f28c180d207b9668c